doc/GUIDE.md

```diff
@@ -1574,13 +1580,36 @@ Selected resolver: lts-3.2
 Wrote project config to: /home/michael/my-yesod-project/stack.yaml
 ```
 
-Alternatively you can use your own templates by specifying the path:
+The default `stack-templates` repository is on [Github](https://github.com),
+under the user account `commercialstack`. You can download templates from a
+different Github user by prefixing the username and a slash:
+
+```
+stack new my-yesod-project yesodweb/yesod-simple
```

Comment:
I think it would be worth specifying exactly how this is resolved: user `yesodweb`, repo `stack-templates`, file `yesod-simple.hsfiles`.

Reply:
Yeah, that sounds good.